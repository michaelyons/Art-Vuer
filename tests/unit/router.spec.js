import { shallowMount } from "@vue/test-utils";
import Router from "@/router.js";

describe("Router.js", () => {
  it("matches snapshot", () => {
    const wrapper = shallowMount(Router);
    expect(wrapper).toMatchSnapshot();
  });
});
