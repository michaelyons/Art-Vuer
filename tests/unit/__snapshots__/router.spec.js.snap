// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router.js matches snapshot 1`] = ``;
